version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "5.1.2-SNAPSHOT"
+ThisBuild / version := "5.1.2-SNAPSHOT"
```

Comment:
Is this change compatible with the sbt-release-plugin? Maybe we can just assume it is and try to deploy and a new version to find out.

Reply:
hard to tell from its website, to be honest. And it doesn't seem to offer a dry-run feature. But since this has been introduce in SBT 1.1 already I would be surprised if it didn't work